test/adapters/file/file.spec.js

```diff
@@ -10,7 +10,7 @@ var expect_to_fail = juttle_test_utils.expect_to_fail;
 
 var tmp_file = tmp.tmpNameSync();
 
-var validFormats = {
+var symmetricalFormats = {
```

Comment:
because grok isn't "symmetrical" and yet doesn't mean its not part of the valid formats ? I can move this back to the original name.